tests/validations/mta/mtaValidations.spec.ts

```diff
@@ -152,15 +205,25 @@ describe("mtaValidations", () => {
     });
 
     it("returns no diagnostics when mta.yaml and dev.mtaext have no errors in the WS", async () => {
-      sinon.stub(fsExtra, "pathExists").resolves(true);
+      stubPathExists(
+        { path: mtaYamlPath, resolvesTo: true },
+        { path: devMtaExtPath, resolvesTo: true }
+      );
       sinon.stub(Mta.prototype, "validate").resolves({});
 
       await testValidateWsMtaYamls([]);
     });
 
     it("returns diagnostics when mta.yaml and dev.mtaext have errors in the WS", async () => {
-      const expectedEntries = prepDiagnostics();
+      const [
+        expectedEntries,
+        getValidationSpy,
+      ] = prepValidationAndDiagnostics();
       await testValidateWsMtaYamls(expectedEntries);
+
+      expect(getValidationSpy.callCount).to.equal(1);
+      expect(getValidationSpy.withArgs(modulePath, [devMtaExtPath]).called).to
```

Comment:
Please use `expect(getValidationSpy.firstCall.args).to.deep.equal([modulePath, [devMtaExtPath]]);` to have a good error message in case of failure